src/utilities/getComputedStyleValue.ts

```diff
@@ -0,0 +1,22 @@
+// https://stackoverflow.com/a/62557418/3511012
+function globalExists(varName: string): boolean {
+  const globalEval = eval
+  try {
+    globalEval(varName)
+    return true
+  } catch {
+    return false
+  }
+}
+
+export function getComputedStyleValue(element: Element, property: string): string | undefined {
+  if (globalExists('window')) {
+    try {
+      return window
+        .getComputedStyle(element, null)
+        .getPropertyValue(property)
+    } catch {
+    // do nothing
+    }
+  }
+}
```

Comment:
I considered using this to get actual container size minus padding, but it ended up pretty complex. In my use case I just make sure the parent of the children didn't have any padding.